ViewModel/ImpactAreaScenario/Editor/SpecificIASEditorVM.cs

```diff
@@ -516,7 +516,7 @@ public void MyMessageHandler(object sender, MessageEventArgs e)
             if (e.Message is FrequencyDamageMessage damageMessage)
             {
                 //todo: not sure that this is correct. Maybe we want the "total" one, but in the current case the "total" has no values?
-                if(e.Message.Message.Equals("Damage-frequency function for damage and asset categoriesdefaultandunassigned"))
+                if(e.Message.Message.Equals("FrequencyDamage"))
```

Comment:
you can use damageMessage.Message instead of e.Message.Message.

Reply:
Is there a constant for FrequencyDamage somewhere that you can reference?

Reply:
I don't think so. This is a message that comes out of the compute, so this string is defined somewhere in HEC model. The message has changed a couple of times. I think HEC has enough on their plate right now, but ideally this would be handled better.

Reply:
i think if you cast, and then use the FrequencyDamageMessage you have access to a const rather than parsing the message string.

Reply:
https://github.com/HydrologicEngineeringCenter/fda-model/blob/597a1a24bbc7a778a6e0b186d477bcd8a4b0528f/fda-model/compute/FrequencyDamageMessage.cs#L21

then dig into the paired data object metadata